src/UID2.Client/UID2Encryption.cs

```diff
@@ -435,5 +441,20 @@ private static IdentityScope DecodeIdentityScopeV3(byte value)
         {
             return (IdentityScope)((value >> 4) & 1);
         }
+
+        private static IdentityType? GetIdentityType(String idType)
+        {
+            switch (idType)
+            {
+                case "A": case "E":
```

Comment:
Curious, why do we do it this way? Didn't we figure out identitytype during decryption?

Also can we please reference the confluence title where these prefixes are specified?

Reply:
I'm also a bit confused about this as well. Isn't there a single bit that indicates this in the UID2?